src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -120,30 +120,40 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
           changes.children = res.map((child, idx) => ({ item_id: child.id, order: idx }));
         }
         if (!this.initialFormData) return throwError(new Error('Invalid form'));
+        if (!Object.keys(changes).length) return of(undefined);
         return this.updateItemService.updateItem(this.initialFormData.id, changes);
       }),
     );
   }
 
   // Item string changes
   private getItemStringChanges(): ItemStringChanges | undefined {
-    const title = this.itemForm.get('title');
-    const subtitle = this.itemForm.get('subtitle');
-    const description = this.itemForm.get('description');
+    const titleControl = this.itemForm.get('title');
+    const subtitleControl = this.itemForm.get('subtitle');
+    const descriptionControl = this.itemForm.get('description');
+    const initialValues = this.initialFormData?.string;
 
-    if (title === null || description === null || subtitle === null) return undefined;
+    if (titleControl === null || subtitleControl === null || descriptionControl === null || !initialValues) return undefined;
 
-    return {
-      title: (title.value as string).trim(),
-      subtitle: (subtitle.value as string).trim() || null,
-      description: (description.value as string).trim() || null,
-    };
+    const res: ItemStringChanges = {};
+
+    const title = titleControl.value as string;
+    if (title !== initialValues.title) res.title = title.trim();
+
+    const subtitle = subtitleControl.value as string;
+    if (subtitle !== initialValues.subtitle) res.subtitle = subtitle.trim();
+
+    const description = descriptionControl.value as string;
+    if (description !== initialValues.description) res.description = description.trim();
+
```

Comment:
Don't you loose the fact that when string is empty for subtitle and description, you were pushing "null" ?

Reply:
Fixed

Reply:
👍  (It was a check you wrote a few weeks ago after many iterations. It should not be me who makes you notice that you do not handle this anymore.)